tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -152,7 +152,10 @@ def confirm_topics_on_all_brokers(self, expected_topic_set):
     def setup_system(self, start_processor=True, num_threads=3, group_protocol='classic'):
         # Setup phase
         use_streams_groups = True if group_protocol == 'streams' else False
-        self.kafka = KafkaService(self.test_context, num_nodes=self.replication, zk=None, topics=self.topics, use_streams_groups=use_streams_groups)
+        self.kafka = KafkaService(self.test_context, num_nodes=self.replication, zk=None, topics=self.topics, server_prop_overrides=[
+            ["offsets.topic.num.partitions", self.partitions],
```

Comment:
offsets.topic.num.partitions actually now defaults to 3 in system tests (may have been 50 at some point - this change introduced the problem). 

It makes sense to not use 50 partitions in a system test. The important thing is that we are consistent and don't resize the topic.